cmd/kube-scheduler/app/server.go

```diff
@@ -207,7 +207,11 @@ func Run(ctx context.Context, cc *schedulerserverconfig.CompletedConfig, sched *
 	if cc.DynInformerFactory != nil {
 		cc.DynInformerFactory.WaitForCacheSync(ctx.Done())
 	}
-
+	// Wait for all handlers to sync (all items in the initial list delivered) before scheduling.
+	if err := sched.WaitForHandlersSync(ctx); err != nil {
+		logger.V(2).Error(err, "error occurred during WaitForHandlersSync")
```

Comment:
In general, errors should not have a verbosity level.

```suggestion
		logger.Error(err, "Failed waiting for handlers to sync")
```

Reply:
You missed the correction in the message. Messages in log lines (unlike errors) start with capital https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message:~:text=Start%20from%20a%20capital%20letter.